views/home.html

```diff
@@ -6,6 +6,11 @@
         {{{localized.tagline}}}
       </p>
     </div>
+    <div>
+      <a href="http://www.covalenceconf.com">
```

Comment:
Can be here used `https://`? This url very stands out on main page (start of page). For someone this can be security issue and no one wants to see `Not secure` label aware url.

(This request based on that, what user require for adding app to catalog)